packages/documentation/src/stories/getting-started/getting-started.docs.mdx

```diff
@@ -0,0 +1,135 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as GettingStartedStories from './getting-started.stories';
+
+<Meta of={GettingStartedStories} />
+
+# Getting started
+
+Welcome to the Swiss post design system world!
+
+## Which package do I need?
+
+The design-system is formed with several packages and different compatibilities with your favorite framework.
+
+<div role="alert" className="alert alert-info">
+  All components require at least the <a href="/?path=/docs/e53e2de8-0bbf-4f70-babc-074c5466f700--docs"><span aria-hidden="true">📦 </span>Styles package</a>.
+</div>
+
+<div className="table-responsive">
+  <table className="table">
+    <caption className="visually-hidden">Compatibilities between components and frameworks and which package is needed</caption>
+    <colgroup>
+      <col style={{width: '25%'}}/>
+      <col style={{width: '25%'}}/>
+      <col style={{width: '25%'}}/>
+      <col style={{width: '25%'}} />
+    </colgroup>
+    <thead>
+      <tr>
+        <th scope="col">Component</th>
+        <th scope="col">JavaScript</th>
+        <th scope="col">Angular</th>
+        <th scope="col">React</th>
```

Comment:
The react package is not published yet.

Reply:
Yes, we can discuss about it. In my opinion, there is an intention to publish so that why I wrote To Be Defined/Decided, and it seems so team at the post are interested or will use react in the near future, so I guess it will interested them?